R/server.R

```diff
@@ -551,6 +551,28 @@ runApp <- function(appDir=getwd(),
     handlerManager$clear()
   }, add = TRUE)
 
+  appParts <- as.shiny.appobj(appDir)
```

Comment:
I'm not 100% sure, but I don't think it's safe to move this line up here. This probably merits a discussion with @jcheng5.